packages/components/cascader/src/cascader.vue

```diff
@@ -294,9 +294,7 @@ const presentTags: Ref<Tag[]> = ref([])
 const allPresentTags: Ref<Tag[]> = ref([])
 const suggestions: Ref<CascaderNode[]> = ref([])
 
-const cascaderStyle = computed<StyleValue>(() => {
-  return attrs.style as StyleValue
-})
+const cascaderStyle = computed<StyleValue>(() => attrs.style as StyleValue)
```

Comment:
We should modify the code in a PR only related to the current problem to be solved. Other irrelevant changes can easily distract attention. If you want to change it, it's best to put it in a new refactoring PR.